### ISSUE TITLE
BUG: Mask should not fail if no sites are to be masked

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/augur#946]
### Current Behavior
When mask is told to mask nothing, it throws an error and doesn't exit successfully.

### Expected behavior
It is perfectly legal to mask nothing, e.g. by passing `--mask-from-beginning 0`

### How to reproduce
```bash
 augur mask --sequences results/global/aligned.fasta --mask-from-beginning 0 --mask-from-end 0 --output results/global/masked.fasta
        
No masking sites provided. Must include one of --mask, --mask-from-beginning, --mask-from-end, --mask-invalid, or --mask-sites
```

Augur 15.0.2

This type of behaviour makes it harder to make generally parametrized workflows. One then needs to check first whether mask has anything to mask or not. Rather than just being able to choose masking from 0 to all continuously without extra logic.

**Comment by j23414** [comment]
Makes sense. Help statement indicates a 0 value should be valid [augur/mask.py#L172](https://github.com/nextstrain/augur/blob/master/augur/mask.py#L172) and as far as I can tell shouldn't break the `mask_sequence` function in [augur/mask.py#L78](https://github.com/nextstrain/augur/blob/15dfc8bc5c5a570066ddd5601c9aa82e0c1d2b7d/augur/mask.py#L78)


Seems to be thrown by a `if not any` statement in [augur/mask.py#L206](https://github.com/nextstrain/augur/blob/master/augur/mask.py#L206)

Which could be changed to a `if not any (i is not None in i in (mask args)`:

```
if not any(i is not None for i in (args.mask_file, args.mask_from_beginning, args.mask_from_end, args.mask_sites, args.mask_invalid)):
```

Smaller example:

```
mask_from_beginning=0 
other_param=None

if not any(i is not None for i in (mask_from_beginning, other_param)):
  print("No")
else:
  print("Yes")
```

Someone else may have better context for why it's throwing an error on 0 values. I haven't looked at augur much before now...

**Comment by jameshadfield** [comment]
My expected behavior is that at least one masking argument should have to be provided, but the value can be zero / an empty file.
